anyway/config.py

```diff
@@ -8,6 +8,8 @@
 # variables
 #
 
+SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL')
+APP_SECRET_KEY = os.environ.get('APP_SECRET_KEY')
```

Comment:
How about using the `parsers.secrets` module?

Reply:
@elazarg do you mean [this module](https://docs.python.org/3/library/secrets.html)?

Reply:
I mean [this module](https://github.com/hasadna/anyway/blob/dev/anyway/parsers/secrets.py).

Reply:
Yes, but I suggest we'll do it in another pr

Reply:
@elazarg actually I added it - did I use it correctly?
https://github.com/hasadna/anyway/pull/1327/commits/f22978fc9c7877ae72bb5bcd672c0f508fb6f119

Reply:
Yes, though of course if you can test that it'd be better.